### ISSUE TITLE
bundled package is not compatible with Angular 5

### ISSUE THREAD

**Issue by e-cloud** [dherges/ng-packagr#258]
## Type of Issue

```
[ ] Bug Report
[x] Feature Request
```

## Description

The metadata version is upgraded to 4 since angular 5, but ng-packagr is generating version 3 metadata, it seems that ng-packagr is not using angular/compiler-cli to compile source.

#### How To Reproduce

_please include instructions how to reproduce / for features: do you have a first draft or idea how to implement?_

#### Expected Behaviour

_please describe what behaviour or result you expected_

#### Version Information

```
ng-packagr: v1.x.y
node: v8.x.y
@angular: v4.x.y
rxjs:
zone.js:
```

_please include any version information that might be relevant, e.g. other third-party libraries_

**Comment by e-cloud** [comment]
see angular/angular#19607

**Comment by dherges** [comment]
Can you check whether the issue persists with the [1.6.0-rc.0](https://github.com/dherges/ng-packagr/blob/master/CHANGELOG.md#160-rc0-2017-11-10)?

It now generates v4 metadata ... and the test coverage for v5 needs improvement. ;_)

**Comment by e-cloud** [comment]
@dherges good, i will give it a try when i get some time.

**Comment by dherges** [comment]
Ok, the metadata version is now 4 and ng-packagr now uses compiler-cli. I'll close this. Should the error re-occure please comment! Or re-open!

**Comment by dherges** [comment]
Fixed by #271
